compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -819,9 +775,8 @@ pub mod debuginfo {
     /// LLVMRustDebugNameTableKind
     #[derive(Clone, Copy)]
     #[repr(C)]
-    pub enum DebugNameTableKind {
+    pub(crate) enum DebugNameTableKind {
         Default,
-        Gnu,
```

Comment:
This is another case where removing unused enum variants is a net negative. It just makes auditing harder.

Reply:
In this case, it was removed from both the Rust and C++ side: would you rather that I add it back to both?

Reply:
Yeah, please add it back to both.

(Our `LLVMRustDebugNameTableKind` enum is basically just a substitute for LLVM's `DICompileUnit::DebugNameTableKind`. It's needed to work around the fact that we can't reliably pass LLVM's C++ enums across FFI, because the LLVM C++ API isn't stable, so upgrading LLVM would potentially introduce silent UB if the enums change incompatibly.)

Reply:
Done